multiqc/modules/hicpro/hicpro.py

```diff
@@ -0,0 +1,386 @@
+
+    def parse_hicpro_stats(self, f):
+        """ Parse a HiC-Pro stat file """
+        s_name = self.clean_s_name(os.path.basename(f['root']), os.path.dirname(f['root']))
+
+        if s_name not in self.hicpro_data.keys():
+            self.hicpro_data[s_name] = {}
+
+        self.add_data_source(f, s_name)
+        for l in f['f'].splitlines():
+            if not l.startswith('#'):
+                s = l.split("\t")
+                if s[0] in self.hicpro_data[s_name]:
+                    log.debug("Duplicated keys found! Overwriting: {}".format(s[0]))
+                self.hicpro_data[s_name][s[0]] = int(s[1])
+
+    def hicpro_stats_table(self):
+         """ Add HiC-Pro stats to the general stats table """
+         headers = OrderedDict()
+         headers['total_R1'] = {
+             'title': 'Total',
+             'description' : 'Total Number of Read Pairs',
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': False
+         }
+
+         headers['mapped_R1'] = {
+             'title': 'Aligned [R1]',
+             'description' : 'Total number of aligned reads [R1] ({})'.format(config.read_count_desc),
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': True
+         }
+
+         headers['percent_mapped_R1'] = {
+             'title': '% Aligned [R1]',
+             'description': 'Percentage of aligned reads [R1] (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+
+         headers['mapped_R2'] = {
+             'title': 'Aligned [R2]',
+             'description' : 'Total number of aligned reads [R2] ({})'.format(config.read_count_desc),
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': True
+         }
+
+         headers['percent_mapped_R2'] = {
+             'title': '% Aligned [R2]',
+             'description': 'Percentage of aligned reads [R2] (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+
+         headers['paired_reads'] = {
+             'title': 'Reported Read Pairs',
+             'description' : 'Total number of reported read pairs ({})'.format(config.read_count_desc),
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': True
+         }
+
+         headers['percent_paired_reads'] = {
+             'title': '% Reported Read Pairs',
```

Comment:
Maybe just `% Reported` to keep it shorter (description can specify that it's read pairs). Also, I'm not totally sure what reported means here..? Is this the percentage of reads left after filtering? Could be good to bulk up the description a little.

Reply:
yes reported means 'both R1/R2 mapped' + pass the users filters in term of multi-hits, mapQ, etc.

Reply:
I changed the description as 'Total number of read pairs ({}) passing the mapping filters'

Reply:
Perfect ✨